### PR TITLE
(CEM-2503) VoiceButton Component

### PR DESCRIPTION
**Pull request by ericlin11354**
CEM-2503 Working on implementing VoiceButton

PR Draft exists to share code with Cat

### CODE REVIEW

**Comment by ralph-dev** [comment]
Also remember to attach a photo of the component to your PR

**Comment by ericlin11354** [comment]
Video of component for visualization.
https://user-images.githubusercontent.com/50276107/136712689-a1107294-414d-4a20-9a33-3fd6003974ce.mp4

**Comment by Curtain64** [comment]
screenshot1
screenshot2
We don't really know how to make it so that the color green would only be within the microphone icon but this is our process thus far.

**Comment by ralph-dev** [comment]
Try reading this: https://developer.mozilla.org/en-US/docs/Web/CSS/mask

File: rollup.config.js [hunk 1]
```diff
@@ -74,5 +74,9 @@ export default {
     external: [
         ...Object.keys(pkg.dependencies || {}),
         ...Object.keys(pkg.peerDependencies || {}),
+<<<<<<< HEAD
```
Comment (ralph-dev) [inline rollup.config.js:77]: Need to fix merge conflicts before merging

**Review by ralph-dev — CHANGES_REQUESTED** [review]

**Comment by Curtain64** [comment]
screenshot3
Our screenshot for our VoiceButton

**Pull request merged**